library/core/src/num/nonzero.rs

```diff
@@ -895,3 +895,39 @@ macro_rules! nonzero_unsigned_is_power_of_two {
 }
 
 nonzero_unsigned_is_power_of_two! { NonZeroU8 NonZeroU16 NonZeroU32 NonZeroU64 NonZeroU128 NonZeroUsize }
+
+macro_rules! nonzero_min_max {
+    ( $( $MinVal:expr , $Ty: ident($Int: ty); )+ ) => {
+        $(
+            impl $Ty {
+                #[unstable(feature = "nonzero_max", issue = "89065")]
```

Comment:
Don't see a reason to use two separate feature gates for min and max.

Reply:
Agreed. 591b89dbd58d890f2e13fadbcfc78f590ec5da4b